sha3/src/macros.rs

```diff
@@ -613,5 +613,12 @@ macro_rules! impl_cshake {
 
         #[cfg(feature = "zeroize")]
         impl ZeroizeOnDrop for $reader {}
+
+        impl CustomizedInit for $name {
+            #[inline]
+            fn new_customized(customization: &[u8]) -> Self {
+                Self::new(customization)
```

Comment:
We probably want to remove the inherent method and move its implementation to this trait impl.

Reply:
I'm not sure to understand how to do that. You meant to move the trait implementation outside of the macro ?

Reply:
I mean remove the inherent method on lines 443-446 and use `Self::new_with_function_name` in the trait impl instead.

Reply:
Ah yes, I think it is done now.